### ISSUE TITLE
Update BUILD.bazel codegen to include `edition = "2018"`

### ISSUE THREAD

**Issue by ParkMyCar** [dropbox/pb-jelly#55]
This is pretty simple, in the codegen script we need to update the template for the `rust_library` bazel rule include `edition = "2018"`, e.g. the rule should be:
```
rust_library(
    name = "{crate}",
    crate_type = "lib",
    edition = "2018",
)
```

**Comment by RSMuthu** [comment]
Hi, I shall take it up.

**Comment by ParkMyCar** [comment]
Great! Thanks @RSMuthu

**Comment by RSMuthu** [comment]
@ParkMyCar  can you please assign this to me

**Comment by ParkMyCar** [comment]
This was closed by #57
